src/earn/PoolCard.tsx

```diff
@@ -53,41 +56,40 @@ export default function PoolCard({ pool, testID = 'PoolCard' }: { pool: Pool; te
           <Text style={styles.keyText}>{t('earnFlow.poolCard.rate')}</Text>
           <Text style={styles.valueTextBold}>
             {t('earnFlow.poolCard.apy', {
-              apy: new BigNumber(pool.apy).multipliedBy(100).toFixed(2),
+              apy: new BigNumber(
+                pool.yieldRates.reduce((acc, yieldRate) => acc + yieldRate.percentage, 0)
+              ).toFixed(2),
             })}
           </Text>
         </View>
         <View style={styles.keyValueRow}>
           <Text style={styles.keyText}>{t('earnFlow.poolCard.reward')}</Text>
           <Text
             style={styles.valueText}
-          >{`${new BigNumber(pool.reward).multipliedBy(100).toFixed(2)}%`}</Text>
+          >{`${pool.earnItems.reduce((acc, earnItem) => acc.plus(new BigNumber(earnItem.amount)), new BigNumber(0)).toFixed(2)}%`}</Text>
```

Comment:
move these computations into a variable to make this more readable and potentially a useMemo